src/AbstractServer.php

```diff
@@ -40,10 +40,10 @@ public function getFunctions(): Definition
      *
      * @deprecated Since 2.7.0; method will have private visibility starting in 3.0.
      */
-    // @codingStandardsIgnoreStart
+    // phpcs:disable
     protected function _buildCallback(Reflection\AbstractFunction $reflection): Method\Callback
```

Comment:
Follow the guidelines in the docblock here, and you can then get rid of the `phpcs:disable`/`phpcs:enable` dance. :wink: (make sure you also update any calls to the method)

Honestly, I'd likely add `private function buildCallback()` in v2, and have `_buildCallback()` proxy to it. Since it's an internal method, we can do this safely, and `_buildCallback()` (a) could be marked `@deprecated`, and (b) would never be called anyways, making overriding it by end-users moot. Then for 3.0, we just remove the underscore-prefixed version.

I'd do that for all underscore-prefixed methods except `_buildSignature()` and `_dispatch()` (which are the only methods extensions will consume).

That would, of course, mean having a new minor version before we merge this particular PR. But I think that it's a worthwhile exercise.

Reply:
If I haven't missed anything, everything mentioned here should be covered. Let me know if further changes are required. Resolving this for now.